dev-docs/adzymic.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: bidder
+title: Adzymic
+description: Adzymic Bidder Adapter
+biddercode: adzymic
+pbjs: true
+pbs: false
+aliasCode: appnexus
+gdpr_supported: true
+media_types: banner, video, native
```

Comment:
Please add the following meta fields

- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
- safeframes_ok: true/false
- 
to help others vet your adapter